inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/InitializerSpec.kt

```diff
@@ -38,17 +40,16 @@ class InitializerSpec : BaseTest() {
 
     @Test
     fun `should add host app info with basic attributes`() {
-        WorkManagerTestInitHelper.initializeTestWorkManager(context!!)
-        InAppMessaging.initialize(context, true)
-        InAppMessaging.instance().registerPreference(TestUserInfoProvider())
-        HostAppInfoRepository.instance().getInAppMessagingSubscriptionKey().shouldNotBeNullOrEmpty()
-        HostAppInfoRepository.instance()
-            .getInAppMessagingSubscriptionKey() shouldBeEqualTo AppManifestConfig(context).subscriptionKey()
-        HostAppInfoRepository.instance()
-            .getPackageName() shouldBeEqualTo "com.rakuten.tech.mobile.inappmessaging.runtime.test"
-        HostAppInfoRepository.instance().getVersion() shouldBeEqualTo "1.0.2"
+        verifyHostAppInfo()
     }
 
+    // API 33 is not yet supported in Robolectric v4.8.1
```

Comment:
nit: Instead of comment, please use @Ignore with explanation annotation

Reply:
updated